### PR TITLE
test: add daily e2e cron job

### PR DESCRIPTION
**Pull request by mmulji-ic**

### CODE REVIEW

**Comment by mmulji-ic** [comment]
@faddat would you add anything else?

File: .github/workflows/nightly-e2e.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+
+  run-tests:
+    uses: cosmos/gaia/.github/workflows/test.yml@main
+
+  run-simulations:
+    uses: cosmos/gaia/.github/workflows/sims.yml@main
```
Comment (glnro) [inline .github/workflows/nightly-e2e.yml]: Just needs a newline

**Review by glnro — COMMENTED** [review]
Technically this is a nightly run of all test suites not just e2e...if the goal is just to isolate e2e then maybe we emulate something more specific a la [tendermint's nightly e2e run](https://github.com/tendermint/tendermint/blob/main/.github/workflows/e2e-nightly-main.yml#L13)

**Comment by mmulji-ic** [comment]
> Technically this is a nightly run of all test suites not just e2e...if the goal is just to isolate e2e then maybe we emulate something more specific a la [tendermint's nightly e2e run](https://github.com/tendermint/tendermint/blob/main/.github/workflows/e2e-nightly-main.yml#L13)

I'm ok running the additional tests as we had non-determinism in the main branch that crept in. Will change the yml file name to reflect thats its not just e2e.

**Pull request merged**